tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
-    "strict": true,
+    "strict": false,
```

Comment:
this creates problems with syntax highlighting as prismjs is a javascript library only.

Reply:
There are typings for PrismJS available which you can install using `yarn add --dev @types/prismjs`. Additionally, you will need to replace the `highlight` function import from this:
```js
import { highlight } from 'prismjs/components/prism-core';
```
to this:
```js
import { highlight } from 'prismjs';
```
because `prismjs/components/prism-core` is an already compiled and minified product of the actual library. NextJS will tree-shake and minify the code for us anyways so it doesn't make any sense to use a cut-down version here.

Interestingly, setting the typings to strict uncovered a real typing issue: The `highlight` function takes three arguments, not two. Your IDE will inform you about this once you fix the typings but you can also see this in the code and documentation of PrismJS: https://github.com/PrismJS/prism/blob/master/prism.js#L660